fdkaac.rc

```diff
@@ -1,3 +1,33 @@
 #include <winresrc.h>
+#include "version.h"
 
 CREATEPROCESS_MANIFEST_RESOURCE_ID RT_MANIFEST "fdkaac.manifest"
+
+1 VERSIONINFO
+//Version-information resource identifier. This value must be 1.
+FILEVERSION VER_VERSION
+PRODUCTVERSION VER_VERSION
+FILEOS 0x4
+//VOS_NT 0x4 File was designed for 32-bit Windows.
+FILETYPE 0x1
+//VFT_APP 0x1 File contains an application.
+
+{
+BLOCK "StringFileInfo"
+{
+	BLOCK "040904E4"
+	{
+		VALUE "FileDescription", "Command line frontend for libfdk-aac encoder"
+		VALUE "FileVersion", VER_VERSION_STR
```

Comment:
I don't know why I can't use that variable `fdkaac_version`. 
```suggestion
		VALUE "FileVersion", fdkaac_version
```
That will cause 
```
$ windres fdkaac.rc -o fdkaac.o
E:\Etc\HUGE\msys64\mingw64\bin\windres.exe: fdkaac.rc:21: syntax error
```